packages/webdriver/src/index.ts

```diff
@@ -154,9 +154,15 @@ export default class WebDriver {
     static async reloadSession(instance: Client, newCapabilities?: WebdriverIO.Capabilities) {
         const capabilities = deepmerge(instance.requestedCapabilities, newCapabilities || {})
         const params: Options.WebDriver = { ...instance.options, capabilities }
+        for (const prop of ['protocol', 'hostname', 'port', 'path'] as string[]) {
```

Comment:
let's better type this so we don't need to use `any`
```suggestion
        for (const prop of ['protocol', 'hostname', 'port', 'path'] as keyof Option.Connection) {
```

Reply:
How about this one?